keps/sig-node/2400-node-swap/README.md

```diff
@@ -426,6 +430,7 @@ _(Tentative.)_
   detects on the host.
 - Consider introducing new configuration modes for swap, such as a node-wide
   swap limit for workloads.
+- Add swap memory to the Kubelet stats api.
```

Comment:
@derekwaynecarr suggested this addition for beta.

@dashpole WDYT?

Reply:
That makes sense to me.  I don't remember what swap metrics are available for cgroups v1 vs v2, but I'd want to at least be able to tell if swap is causing problems at the node level.  It may also be nice to have swap metrics at the pod/container level if the available metrics can tell me if swapping is hurting my application's performance in some way.

Reply:
@dashpole cgroup v2 has `memory.swap.current` which exists on non-root cgroups which is total amount of swap currently being used by the cgroup and its descendants.

It appears to already be supported in cAdvisor, we just did not include it in k8s API for MemoryStats as it was not yet supported to be on.

see: https://github.com/google/cadvisor/blob/ef7e64f9efab1257e297d7af339e94bb016cf221/container/libcontainer/handler.go#L800

Reply:
Looks like cgroups v1 has an equivalent, so we should be all set in that regard